src/library_syscall.js

```diff
@@ -227,7 +227,7 @@ var SyscallsLibrary = {
   },
   __syscall5: function(which, varargs) { // open
     var pathname = SYSCALLS.getStr(), flags = SYSCALLS.get(), mode = SYSCALLS.get() // optional TODO
-    var stream = FS.open(pathname, flags, mode);
+    var stream = FS.open(pathname, flags | {{{ cDefine('O_LARGEFILE') }}}, mode);
```

Comment:
I think the behavior is the same; previously the `open()` wrapper around `sys_open` would always add the flag, so this just changes where the flag is added, right?